src/utils/Cypher.ts

```diff
@@ -2,43 +2,34 @@ import BackendError from '../exception/BackendError';
 import Configuration from './Configuration';
 import Constants from './Constants';
 import CryptoConfiguration from '../types/configuration/CryptoConfiguration';
+import { CryptoSetting } from '../types/Setting';
+import SettingStorage from '../storage/mongodb/SettingStorage';
+import Utils from './Utils';
 import _ from 'lodash';
 import crypto from 'crypto';
 
 const IV_LENGTH = 16;
 const MODULE_NAME = 'Cypher';
 
 export default class Cypher {
-  private static configuration: CryptoConfiguration;
 
-  public static getConfiguration(): CryptoConfiguration {
-    if (!this.configuration) {
-      this.configuration = Configuration.getCryptoConfig();
-      if (!this.configuration) {
-        throw new BackendError({
-          source: Constants.CENTRAL_SERVER,
-          module: MODULE_NAME,
-          method: 'getConfiguration',
-          message: 'Crypto configuration is missing'
-        });
-      }
-    }
-    return this.configuration;
+  public static async getCryptoSetting(tenantID: string): Promise<CryptoSetting> {
+    return (await SettingStorage.getCryptoSettings(tenantID)).crypto;
```

Comment:
Please keep the error throwing if no settings are found. 
E-mobility can't work otherwise and it shall not happen.